### ISSUE TITLE
Pls merge my new wiki pages in the main wiki

### ISSUE THREAD

**Issue by Okazakee** [tomsom/yoga-linux#20]
I have written some wiki docs for vaapi codecs/hw acceleration stuff and other cool things.
@tomsom Please take a look [here](https://github.com/Okazakee/yoga-linux/wiki).

**Comment by rodude123** [comment]
This sounds great but what are the commands for arch I cannot run grubby unless I install some of the fedora grub packages.

**Comment by Okazakee** [comment]
> This sounds great but what are the commands for arch I cannot run grubby unless I install some of the fedora grub packages.

I'll leave this issue open, try to find a way to get the same result, I will immediately insert your findings into the wiki

**Comment by stuarthayhurst** [comment]
Never used grubby, but wouldn't it just modify `GRUB_CMDLINE_LINUX_DEFAULT` in `/etc/default/grub`?

**Comment by Okazakee** [comment]
> Never used grubby, but wouldn't it just modify `GRUB_CMDLINE_LINUX_DEFAULT` in `/etc/default/grub`?

Yes it should work that way, dunno what are the differences, I just found the solution online and inserted it in the wiki

Also, it should be rerun after kernel update

**Comment by stuarthayhurst** [comment]
On my laptop, I just made a file `/etc/modules-load.f/amd_pstate.conf`, containing `amd_pstate`. No changes with grubby, no changes to `/etc/default/grub`. For me, this has worked on every boot without fail. I'm running kernel 6.1-rc5 though as I'd have to recompile again with the patch for my Samsung SSD. Somewhere later in the 6.1 cycle, I think `amd-pstate=passive` is required as a boot parameter, but I can't test this until I'm on a newer kernel.

**Comment by stuarthayhurst** [comment]
The SSD patch is from [this thread](https://lore.kernel.org/all/20221116171727.4083-1-git@augustwikerfors.se/T/), but I've attached it as a file for you: [nvme.patch.txt](https://github.com/tomsom/yoga-linux/files/10484174/nvme.patch.txt)

If you're comfortable compiling your own kernel, include that patch if you're having issues with a Samsung SSD. I'd send you a copy of my patched kernel, but I built it for Debian, and I'm fairly sure you're running Fedora from what you've said?

Anyway, I'm pretty swamped at the moment, but if I get some time, I'll play around with the current linux 6.2 release candidates and see what's required to get it running reliably. There's some upstream work around the EPP part of the pstate driver, so that might take a bit more effort until it's merged, but I'll see what's required to get that running.

**Comment by Okazakee** [comment]
> The SSD patch is from [this thread](https://lore.kernel.org/all/20221116171727.4083-1-git@augustwikerfors.se/T/), but I've attached it as a file for you: [nvme.patch.txt](https://github.com/tomsom/yoga-linux/files/10484174/nvme.patch.txt)
> 
> If you're comfortable compiling your own kernel, include that patch if you're having issues with a Samsung SSD. I'd send you a copy of my patched kernel, but I built it for Debian, and I'm fairly sure you're running Fedora from what you've said?
> 
> Anyway, I'm pretty swamped at the moment, but if I get some time, I'll play around with the current linux 6.2 release candidates and see what's required to get it running reliably. There's some upstream work around the EPP part of the pstate driver, so that might take a bit more effort until it's merged, but I'll see what's required to get that running.

I can't figure out how to patch the kernel in fedora, i got kernel-devel and patch, but dunno what command i should run

**Comment by stuarthayhurst** [comment]
As in you couldn't patch it, the build failed with the patch, or the patch didn't work?
This is probably better on the [issue for the Samsung SSD patch anyway](https://github.com/tomsom/yoga-linux/issues/9)

**Comment by stuarthayhurst** [comment]
https://www.phoronix.com/news/AMD-P-State-Active-Default

It seems the pstate driver will be the default and in active mode soon, so the wiki will probably need updating

Also, I noticed the links are all like https://github.com/tomsom/yoga-linux/wiki/wiki/Installation, instead of https://github.com/tomsom/yoga-linux/wiki/Installation. Having 2 /wiki/s takes you to the edit page, so normal users get redirected and given a banner saying insufficient permissions.

**Comment by Okazakee** [comment]
> https://www.phoronix.com/news/AMD-P-State-Active-Default
> 
> It seems the pstate driver will be the default and in active mode soon, so the wiki will probably need updating
> 
> Also, I noticed the links are all like https://github.com/tomsom/yoga-linux/wiki/wiki/Installation, instead of https://github.com/tomsom/yoga-linux/wiki/Installation. Having 2 /wiki/s takes you to the edit page, so normal users get redirected and given a banner saying insufficient permissions.

Nice news about pstate, for the wiki, i guess @tomsom should fix that as there are no direct ways to PR and merge edits for wikis

**Comment by stuarthayhurst** [comment]
The changed default won't land until kernel 6.5 at the earliest AFAIK, so the change to the GRUB config is still required for 6.3 / 6.4.

I can also confirm the links seem to be working now, thanks
